src/api/checkin.js

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import { addAction } from '../lib/database';
+
+let app = express();
+
+app.post('/', (req, res) => {
```

Comment:
Could you document this new endpoint in the [REST doc](https://github.com/TheFourFifths/consus/blob/dev/doc/REST.md)?